content/docs/addons-test-utils.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: test-utils
-title: Test Utilities
+title: परीक्षण उपयोगिताएँ
 permalink: docs/test-utils.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**आयात करना**
```

Comment:
Lets keep this `इम्पॉर्टिंग`.
`आयात करना` is too complex in this context

Reply:
Done.